README.rst

```diff
@@ -106,22 +106,36 @@ Example:
 
    $ mkdir test && cd test && git init
```

Comment:
tbh, this whole example block is not really formatted in a readable way or consistent with the way i see example blocks in other documentation, perhaps i can offer some suggestions

Reply:
Yes, I would appreciate your improvements!